packages/SystemsThinkingWorkbench-Core.package/STWNode.class/instance/lineWidth..st

```diff
@@ -1,5 +1,10 @@
 accessing
-lineWidth: aNumber
+lineWidth: aNumber 
 	"Provide uniform access for nodes an connections."
-
-	self borderWidth: aNumber.
+	aNumber == STWSimulation hoverLineWidth
+		ifTrue: [
+		((self width - self fillingMorph width < (STWSimulation hoverLineWidth * 2)) and: (self borderWidth ~= STWSimulation hoverLineWidth))
```

Comment:
Maybe we could extract this condition to another method, it might make this more readable.